src/TableContent.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import URI from 'urijs'
+
+import tableHeaderCells from './tableHeaderCells'
+
+const TableContent = ({enableIndex, tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex,
+  ascendingOrder, enableDownload, checkedRows, currentPageData, host, entriesPerPage, currentPage,
+  tableHeaderOnClick, tableHeaderOnChange, downloadOnChange}) =>
+  <div className={`row expanded`}>
+    <div className={`small-12 columns`} >
+      <Table border>
+        <Table.Head>
+          {
+            enableIndex && <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={100} flexGrow={100}>Index</Table.TextHeaderCell>
+          }
+
+          {
+            tableHeaderCells(tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex, ascendingOrder,
+              columnNumber => tableHeaderOnClick(columnNumber),
+              (value, columnNumber) => tableHeaderOnChange(value, columnNumber)
+            )
+          }
+
+          {
+            enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
+              {
+                checkedRows.length > 0 ?
+                  <a href={URI(`${host}experiments/download/zip`).search({accession: checkedRows})}>
```

Comment:
The right way to use URI is `URI(relativeUrl, domainAndPath).toString()`, so this should be:
```javascript
URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()
```

The final `.toString()`  should be added, beacuse URI is an object and sometimes JavaScript isn’t clever enough to coerce it to a string. You’ve been lucky here. :wink: